src/app/schedule-view/component.ts

```diff
@@ -142,6 +144,25 @@ export class ScheduleViewComponent implements OnInit, OnDestroy {
     return this.schedules[this.scheduleIndex].statusText;
   }
 
+  public showExport () : void {
+      this.isExporting = true;
+  }
+
+  public downloadImage () : void {
+      var node = document.getElementById('schedule-container');
```

Comment:
The conventional way to get DOM elements in Angular is using the `ViewChild` API. see: https://angular.io/api/core/ViewChild